notice about nginx_proxy

ttodua opened the pull request:
**Proposed changes:**
it shows `service "nginx_proxy" refers to undefined network frontend: invalid compose project` during `./docker.sh letsencrypt-create` or  `docker-compose up -d` . as [Vaalyn](https://github.com/AzuraCast/AzuraCast/issues/5078#issuecomment-1031365284) noted here, it is no longer needed for rolling versions.

ttodua committed:
notice about nginx_proxy

Otherwise, it shows `service "nginx_proxy" refers to undefined network frontend: invalid compose project` during `./docker.sh letsencrypt-create` or  `docker-compose up -d` .

ttodua commented:
I am not sure why, but this docker-compose https://docs.azuracast.com/en/administration/docker/multi-site-installation#create-docker-composeyml-file doesn't work when you follow the instructions step by step, it throws an error  `network azuracast_frontend declared as external, but could not be found`
might be related to : https://github.com/AzuraCast/AzuraCast/issues/5191#issuecomment-1090465133 and this https://github.com/AzuraCast/AzuraCast/issues/5291#issuecomment-1114299735

ttodua committed:
Update multi-site-installation.md

ttodua committed:
Update multi-site-installation.md

ItsMitchh commented:
Hi there, 

We do have at the top of the docs a notice that the docs is not updated, I believe Silver has plans to work on them when he has some time.

ttodua commented:
> Hi there,
> 
> We do have at the top of the docs a notice that the docs is not updated, I believe Silver has plans to work on them when he has some time.

Yes, I noticed that message, however, even being that message here, it's impossible to install/follow that tutorial successfully, so, as a temporary workaround or slightest help to others (before that docs will be updated by SE) this might help someone, even if not merged.

ttodua committed:
added wp port

in previous versions of azuracast docs, ports were mentioned. in current version, there is no mention of ports, causing similar issues as #5088

ttodua committed:
Update multi-site-installation.md

more details
